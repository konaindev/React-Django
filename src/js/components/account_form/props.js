export const props = {
  steps: [
    { name: "Set Password", isComplete: true },
    { name: "Complete Account", isActive: true }
  ],
  title: "Complete your account",
  subtitle:
    "We need just a bit more information about you to complete your account."
};
