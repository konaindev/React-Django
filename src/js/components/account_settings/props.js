const user = {
  account_id: 1,
  account_name: "Remarkably",
  account_settings_url: "/users/account-settings",
  email: "test@psl.com",
  logout_url: "/users/logout/",
  user_id: "usr_jjzpeyfeshzpaha5"
};

const itemsOrder = ["profile", "lock", "email"];

const rules = [
  {
    label: "Be at least 8 characters",
    key: "length"
  },
  {
    label: "Contain alphabetic characters",
    key: "characters"
  },
  {
    label: "Not match personal information",
    key: "personal"
  },
  {
    label: "Not be a commonly used password",
    key: "used"
  }
];

const company_roles = [
  { label: "Owner", value: "owner" },
  { label: "Developer", value: "developer" },
  { label: "Asset Manager", value: "asset_manager" },
  { label: "Property Manager", value: "property_manager" }
];

const office_options = [
  { label: "Global", value: 1 },
  { label: "National", value: 2 },
  { label: "Regional", value: 3 },
  { label: "Other", value: 4 }
];

const office_countries = [
  { label: "United States of America", value: "USA" },
  { label: "United Kingdom", value: "GBR" }
];

const us_state_list = [
  { label: "Washington", value: "Washington" },
  { label: "New York", value: "New York" }
];

const gb_county_list = [
  { label: "London", value: "London" },
  { label: "Aberdeen", value: "Aberdeen" }
];

const profile = {
  avatar_url:
    "https://lh3.googleusercontent.com/-cQLcFi7r2uc/AAAAAAAAAAI/AAAAAAAAAAA/ACHi3rfoCSVbR8qVruV55uAYdSC-znVn2w.CMID/s96-c/photo.jpg",
  first_name: "Phillip",
  last_name: "McPhillipson",
  title: "Founder",
  phone_country_code: "",
  phone: "",
  phone_ext: "",
  company: { label: "Glacier Associates", value: "123456789" },
  company_roles: ["owner", "asset_manager", "property_manager"],
  office_street: "1730 Minor Avenue",
  office_city: "Lansing",
  office_state: { label: "Michigan", value: "Michigan" },
  office_zip: "11111",
  office_country: { label: "United States of America", value: "USA" },
  office_address: "1730 Minor Avenue, Lansing, MI",
  office_name: "Michigan",
  office_type: 3
};

const tabsOrder = ["portfolio", "group", "property"];

const initialTab = "portfolio";

export const props = {
  rules,
  profile,
  user,
  company_roles,
  office_options,
  itemsOrder,
  tabsOrder,
  initialTab,
  office_countries,
  us_state_list,
  gb_county_list,
  dispatch: () => {}
};
