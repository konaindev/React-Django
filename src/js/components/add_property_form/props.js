export const props = {
  post_url: "/projects/new",
  states: [
    { id: "AL", name: "AL" },
    { id: "GA", name: "GA" },
    { id: "NE", name: "NE" },
    { id: "OR", name: "OR" },
    { id: "WY", name: "WY" }
  ],
  packages: [
    { id: "accelerate", name: "Accelerate" },
    { id: "optimize", name: "Optimize" },
    { id: "ground", name: "Ground Up" },
    { id: "other", name: "Not Sure" }
  ]
};
