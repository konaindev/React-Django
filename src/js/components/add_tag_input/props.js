const suggestedTags = [
  { name: "Tag 1", count: 5 },
  { name: "Test Tag 2", count: 3 },
  { name: "Lake Highland Properties 3", count: 2 },
  { name: "New Tag", count: 1 }
];

export default { suggestedTags };
