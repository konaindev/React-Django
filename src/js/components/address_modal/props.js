export default {
  isOpen: true,
  addresses: {
    suggested_address: {
      office_city: "Washington",
      office_state: "DC",
      office_street: "1600 Pennsylvania Ave NW",
      office_zip: "20500",
      office_country: "United States"
    }
  },
  dispatch_type: "FAKE DISPATCH"
};
