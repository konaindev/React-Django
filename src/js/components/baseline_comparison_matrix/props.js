import {
  baselineReport,
  baselineReportWithoutCompetitors,
  baselineReportWithOneCompetitor
} from "../common_report/props";

export const props = { report: baselineReport };
export const no_competitor_props = { report: baselineReportWithoutCompetitors };
export const one_competitor_props = {
  report: baselineReportWithOneCompetitor
};
