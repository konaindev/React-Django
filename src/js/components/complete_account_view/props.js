import PropTypes from "prop-types";

export const props = {
  office_types: [
    { label: "Global", value: "global" },
    { label: "National", value: "national" },
    { label: "Regional", value: "regional" },
    { label: "Other", value: "other" }
  ],
  company_roles: [
    { label: "Owner", value: "owner" },
    { label: "Developer", value: "developer" },
    { label: "Asset Manager", value: "asset_manager" },
    { label: "Property Manager", value: "property_manager" },
    { label: "JV / Investor", value: "investor" },
    { label: "Vendor / Consultant", value: "vendor" }
  ],
  office_countries: [
    { label: "United States of America", value: "USA" },
    { label: "United Kingdom", value: "GBR" }
  ],
  office_addresses: [
    {
      value: "Columbia Tower",
      street: "111 Columbia Street",
      city: "Seattle",
      state: "WA"
    },
    {
      value: "1800 Ocean Front Walk, Venice, CA 90291, USA",
      street: "1800 Ocean Front Walk",
      city: "Los Angeles",
      state: "CA"
    },
    {
      value: "2284 W Commodore Way, Seattle, WA 98199, USA",
      street: "2284 W Commodore Way",
      city: "Seattle",
      state: "WA"
    },
    {
      value: "305 Harrison St, Seattle, WA 98109, USA",
      street: "305 Harrison St",
      city: "Seattle",
      state: "WA"
    },
    {
      value: "6460 Ramirez Mesa Dr, Malibu, CA 90265, USA",
      street: "6460 Ramirez Mesa Dr",
      city: "Malibu",
      state: "CA"
    },
    {
      value: "1800 Ocean Front Walk, Venice, CA 90291, USA",
      street: "1800 Ocean Front Walk",
      city: "Los Angeles",
      state: "CA"
    },
    {
      value: "1202 W Thomas Rd, Phoenix, AZ 85013, USA",
      street: "1202 W Thomas Rd",
      city: "Phoenix",
      state: "AZ"
    }
  ]
};
