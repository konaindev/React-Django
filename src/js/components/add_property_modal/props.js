import { states } from "../add_property_form/states";

export const props = {
  formProps: {
    post_url: null,
    packages: [
      { id: "accelerate", name: "Accelerate" },
      { id: "optimize", name: "Optimize" },
      { id: "ground", name: "Ground Up" },
      { id: "other", name: "Not Sure" }
    ],
    states
  }
};
