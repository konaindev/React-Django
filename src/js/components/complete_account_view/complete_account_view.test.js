import renderer from "react-test-renderer";
import { MemoryRouter } from "react-router-dom";
import { Provider } from "react-redux";
import { createStore } from "redux";

import { CompleteAccountView } from "./index";
import { props } from "./props";

const _ = x =>
  createStore(() => ({
    network: {
      isFetching: false
    }
  }));

describe("CompleteAccountView", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <Provider store={_()}>
          <MemoryRouter>
            <CompleteAccountView {...props} />
          </MemoryRouter>
        </Provider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
