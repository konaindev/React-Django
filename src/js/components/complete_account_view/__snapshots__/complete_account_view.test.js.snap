// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CompleteAccountView renders correctly 1`] = `
<div
  className="page-auth"
>
  <div
    className="page-auth__header"
  >
    <a
      className="page-auth__back-link"
      href="/"
      onClick={[Function]}
      style={
        Object {
          "color": "inherit",
          "textDecoration": "inherit",
        }
      }
    >
      ‚Üê Go to Login
    </a>
    <div
      className="page-auth__logo"
    >
      <a
        href="/dashboard"
        onClick={[Function]}
        style={
          Object {
            "color": "inherit",
            "textDecoration": "inherit",
          }
        }
      >
        <div
          className="remarkably-logo"
        >
           
        </div>
      </a>
    </div>
  </div>
  <div
    className="page-auth__body"
  >
    <div
      className="account-form complete-account"
    >
      <div
        className="wizard-progress account-form__wizard"
      >
        <div
          className="wizard-progress__step wizard-progress__step--complete"
        >
          <div
            className="wizard-progress__step-number"
          >
            1
          </div>
          <svg
            aria-hidden="true"
            className="wizard-progress__icon"
            color="inherit"
            focusable="false"
            height="1em"
            role="presentation"
            viewBox="0 0 14 14"
            width="1em"
          >
            <rect
              fill="white"
              height="7"
              width="6"
              x="4"
              y="3"
            />
            <path
              clipRule="evenodd"
              d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.94652 5.04456L6.87461 9.57471C6.78379 9.70919 6.63884 9.8 6.49564 9.8C6.35243 9.8 6.19526 9.72141 6.09397 9.62187C5.99268 9.52232 4.29169 7.81784 4.29169 7.81784C4.16944 7.69559 4.16944 7.49476 4.29169 7.37251L4.73702 6.92718C4.85926 6.80493 5.0601 6.80493 5.1806 6.92718L6.35418 8.09901L8.90392 4.33727C9.00172 4.19407 9.19906 4.15739 9.34226 4.25345L9.86444 4.60796C10.0059 4.70402 10.0443 4.90136 9.94652 5.04456Z"
              fill="currentColor"
              fillRule="evenodd"
            />
          </svg>
          Set Password
        </div>
        <div
          className="wizard-progress__step wizard-progress__step--active"
        >
          <div
            className="wizard-progress__step-number"
          >
            2
          </div>
          Complete Account
        </div>
      </div>
      <div
        className="account-form__title"
      >
        Complete your account
      </div>
      <div
        className="account-form__subtitle"
      >
        We need just a bit more information about you to complete your account.
      </div>
      <div
        className="account-form__body"
      >
        <form
          onReset={[Function]}
          onSubmit={[Function]}
        >
          <div
            className="complete-account__section-label complete-account__section-label--first"
          >
            General Info
          </div>
          <div
            className="complete-account__field-set account-form__field"
          >
            <div
              className="form-field complete-account__field form-field--without-icon"
            >
              <div
                className="form-field__error"
              />
              <div
                className="form-field__label"
              >
                first name
              </div>
              <div
                className="form-field__input-container"
              >
                <input
                  className="input input--highlight"
                  name="first_name"
                  onBlur={[Function]}
                  onChange={[Function]}
                  type="text"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--error"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 14 14"
                  width="1em"
                >
                  <rect
                    fill="white"
                    height="6"
                    width="6"
                    x="4"
                    y="4"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.42026 8.83969L8.83794 9.42027C8.73162 9.52658 8.55823 9.52658 8.45191 9.42027L6.99939 7.96799L5.54686 9.42027C5.44054 9.52658 5.26715 9.52658 5.16083 9.41864L4.57851 8.83969C4.47383 8.73174 4.47383 8.56002 4.57851 8.45208L6.03104 6.9998L4.58015 5.54915C4.47383 5.44121 4.47383 5.26785 4.58015 5.16154L5.16083 4.58096C5.26715 4.47301 5.44218 4.47301 5.5485 4.58096L6.99939 6.03161L8.45191 4.58096C8.55823 4.47301 8.73325 4.47301 8.83794 4.58096L9.42026 5.15991C9.52658 5.26785 9.52658 5.44121 9.42026 5.54915L7.96774 6.9998L9.42026 8.45208C9.52494 8.56002 9.52494 8.73174 9.42026 8.83969Z"
                    fill="#FF540D"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--ok"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 14 14"
                  width="1em"
                >
                  <rect
                    fill="white"
                    height="7"
                    width="6"
                    x="4"
                    y="3"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.94652 5.04456L6.87461 9.57471C6.78379 9.70919 6.63884 9.8 6.49564 9.8C6.35243 9.8 6.19526 9.72141 6.09397 9.62187C5.99268 9.52232 4.29169 7.81784 4.29169 7.81784C4.16944 7.69559 4.16944 7.49476 4.29169 7.37251L4.73702 6.92718C4.85926 6.80493 5.0601 6.80493 5.1806 6.92718L6.35418 8.09901L8.90392 4.33727C9.00172 4.19407 9.19906 4.15739 9.34226 4.25345L9.86444 4.60796C10.0059 4.70402 10.0443 4.90136 9.94652 5.04456Z"
                    fill="currentColor"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
            <div
              className="form-field complete-account__field form-field--without-icon"
            >
              <div
                className="form-field__error"
              />
              <div
                className="form-field__label"
              >
                Last name
              </div>
              <div
                className="form-field__input-container"
              >
                <input
                  className="input input--highlight"
                  name="last_name"
                  onBlur={[Function]}
                  onChange={[Function]}
                  type="text"
                  value=""
                />
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--error"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 14 14"
                  width="1em"
                >
                  <rect
                    fill="white"
                    height="6"
                    width="6"
                    x="4"
                    y="4"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.42026 8.83969L8.83794 9.42027C8.73162 9.52658 8.55823 9.52658 8.45191 9.42027L6.99939 7.96799L5.54686 9.42027C5.44054 9.52658 5.26715 9.52658 5.16083 9.41864L4.57851 8.83969C4.47383 8.73174 4.47383 8.56002 4.57851 8.45208L6.03104 6.9998L4.58015 5.54915C4.47383 5.44121 4.47383 5.26785 4.58015 5.16154L5.16083 4.58096C5.26715 4.47301 5.44218 4.47301 5.5485 4.58096L6.99939 6.03161L8.45191 4.58096C8.55823 4.47301 8.73325 4.47301 8.83794 4.58096L9.42026 5.15991C9.52658 5.26785 9.52658 5.44121 9.42026 5.54915L7.96774 6.9998L9.42026 8.45208C9.52494 8.56002 9.52494 8.73174 9.42026 8.83969Z"
                    fill="#FF540D"
                    fillRule="evenodd"
                  />
                </svg>
                <svg
                  aria-hidden="true"
                  className="form-field__icon form-field__icon--ok"
                  color="inherit"
                  focusable="false"
                  height="1em"
                  role="presentation"
                  viewBox="0 0 14 14"
                  width="1em"
                >
                  <rect
                    fill="white"
                    height="7"
                    width="6"
                    x="4"
                    y="3"
                  />
                  <path
                    clipRule="evenodd"
                    d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.94652 5.04456L6.87461 9.57471C6.78379 9.70919 6.63884 9.8 6.49564 9.8C6.35243 9.8 6.19526 9.72141 6.09397 9.62187C5.99268 9.52232 4.29169 7.81784 4.29169 7.81784C4.16944 7.69559 4.16944 7.49476 4.29169 7.37251L4.73702 6.92718C4.85926 6.80493 5.0601 6.80493 5.1806 6.92718L6.35418 8.09901L8.90392 4.33727C9.00172 4.19407 9.19906 4.15739 9.34226 4.25345L9.86444 4.60796C10.0059 4.70402 10.0443 4.90136 9.94652 5.04456Z"
                    fill="currentColor"
                    fillRule="evenodd"
                  />
                </svg>
              </div>
            </div>
          </div>
          <div
            className="form-field account-form__field form-field--without-icon"
          >
            <div
              className="form-field__error"
            />
            <div
              className="form-field__label"
            >
              title (Optional)
            </div>
            <div
              className="form-field__input-container"
            >
              <input
                className="input input--highlight"
                name="title"
                onBlur={[Function]}
                onChange={[Function]}
                type="text"
                value=""
              />
              <svg
                aria-hidden="true"
                className="form-field__icon form-field__icon--error"
                color="inherit"
                focusable="false"
                height="1em"
                role="presentation"
                viewBox="0 0 14 14"
                width="1em"
              >
                <rect
                  fill="white"
                  height="6"
                  width="6"
                  x="4"
                  y="4"
                />
                <path
                  clipRule="evenodd"
                  d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.42026 8.83969L8.83794 9.42027C8.73162 9.52658 8.55823 9.52658 8.45191 9.42027L6.99939 7.96799L5.54686 9.42027C5.44054 9.52658 5.26715 9.52658 5.16083 9.41864L4.57851 8.83969C4.47383 8.73174 4.47383 8.56002 4.57851 8.45208L6.03104 6.9998L4.58015 5.54915C4.47383 5.44121 4.47383 5.26785 4.58015 5.16154L5.16083 4.58096C5.26715 4.47301 5.44218 4.47301 5.5485 4.58096L6.99939 6.03161L8.45191 4.58096C8.55823 4.47301 8.73325 4.47301 8.83794 4.58096L9.42026 5.15991C9.52658 5.26785 9.52658 5.44121 9.42026 5.54915L7.96774 6.9998L9.42026 8.45208C9.52494 8.56002 9.52494 8.73174 9.42026 8.83969Z"
                  fill="#FF540D"
                  fillRule="evenodd"
                />
              </svg>
              <svg
                aria-hidden="true"
                className="form-field__icon form-field__icon--ok"
                color="inherit"
                focusable="false"
                height="1em"
                role="presentation"
                viewBox="0 0 14 14"
                width="1em"
              >
                <rect
                  fill="white"
                  height="7"
                  width="6"
                  x="4"
                  y="3"
                />
                <path
                  clipRule="evenodd"
                  d="M0 7C0 3.13506 3.13506 0 7 0C10.8649 0 14 3.13506 14 7C14 10.8649 10.8649 14 7 14C3.13506 14 0 10.8649 0 7ZM9.94652 5.04456L6.87461 9.57471C6.78379 9.70919 6.63884 9.8 6.49564 9.8C6.35243 9.8 6.19526 9.72141 6.09397 9.62187C5.99268 9.52232 4.29169 7.81784 4.29169 7.81784C4.16944 7.69559 4.16944 7.49476 4.29169 7.37251L4.73702 6.92718C4.85926 6.80493 5.0601 6.80493 5.1806 6.92718L6.35418 8.09901L8.90392 4.33727C9.00172 4.19407 9.19906 4.15739 9.34226 4.25345L9.86444 4.60796C10.0059 4.70402 10.0443 4.90136 9.94652 5.04456Z"
                  fill="currentColor"
                  fillRule="evenodd"
                />
              </svg>
            </div>
          </div>
          <div>
            <div
              className="complete-account__section-label"
            >
              Company Info 
            </div>
            <div
              className="button button--div button--secondary-gray complete-account__edit-button"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex="0"
            >
              Enter Company info
            </div>
          </div>
          <div>
            <div
              className="complete-account__section-label"
            >
              Office Info
            </div>
            <div
              className="button button--div button--secondary-gray complete-account__edit-button"
              onClick={[Function]}
              onKeyPress={[Function]}
              tabIndex="0"
            >
              Enter Office info
            </div>
          </div>
          <div
            className="complete-account__terms"
          >
            <div
              className="checkbox complete-account__checkbox"
              onClick={[Function]}
            />
            <input
              checked={false}
              hidden={true}
              name="terms"
              readOnly={true}
              type="checkbox"
            />
            Accept¬†
            <a
              className="complete-account__link"
              href="https://www.remarkably.io/terms"
              target="_blank"
            >
              Terms and Conditions
            </a>
          </div>
          <button
            className="button button--block button--uppercase button--disabled-light complete-account__button"
            onClick={[Function]}
            type="submit"
          >
            complete account
          </button>
        </form>
      </div>
    </div>
  </div>
  <div
    className="page-auth__footer"
  >
    ¬© 2019 Remarkably. All Rights Reserved
  </div>
</div>
`;
